org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/ServerInfo.java

```diff
@@ -80,14 +83,15 @@ public String toString() {
    *  - Java : (path to java.home])
    *  - Git : ([Branch] short commit id - commit message)
    * </pre>
-   * 
+   *
    * @return the formatted server details
    */
   public String details() {
     StringBuilder details = new StringBuilder();
     details.append("LemMinX Server info:");
     append(details, "Version", getVersion());
     append(details, "Java", getJava());
+    append(details, "Native Image", Boolean.toString(IS_NATIVE_IMAGE));
```

Comment:
```java
if (IS_NATIVE_IMAGE) {
  append(details, "Native Image", null);
} else {
  append(details, "Java", getJava());
}
```

We might also display graalvm/substratevm version instead

Reply:
I added a "VM Version" entry that will show the `java.vm.version`, which will show the GraalVM version when running as a native-image, or the Java version if using a JDK/JRE.